### ISSUE TITLE
Suggestion RE: Fainted Pokemon

### ISSUE THREAD

**Issue by fawfulfan** [SheerSt/pokewilds#346]
This isn't so much a bug report as it is a suggestion for the game mechanics.

Fainted Pokemon should not be able to perform field actions or follow the player. I know that fainted Pokemon can do this in the main series games, but that never really made sense to me, and given that this is a survival/exploration game that kind of does its own thing, I think disabling all actions for a fainted Pokemon would be more realistic and a refreshing twist on the standard game mechanics. In particular it would give the player a lot more incentive to strategize, make use of the sleeping bag, and make safe clearings here and there to protect themselves.

At the very least, this should be an option players can select while starting the game, for added difficulty.

**Comment by TinchoX** [comment]
> This isn't so much a bug report as it is a suggestion for the game mechanics.
> 
> Fainted Pokemon should not be able to perform field actions or follow the player. I know that fainted Pokemon can do this in the main series games, but that never really made sense to me, and given that this is a survival/exploration game that kind of does its own thing, I think disabling all actions for a fainted Pokemon would be more realistic and a refreshing twist on the standard game mechanics. In particular it would give the player a lot more incentive to strategize, make use of the sleeping bag, and make safe clearings here and there to protect themselves.
> 
> At the very least, this should be an option players can select while starting the game, for added difficulty.

Agreed, it makes the game much more interesting as well, just having fainted mon and using them like nothing takes the fun out of it.

**Comment by fawfulfan** [comment]
> > This isn't so much a bug report as it is a suggestion for the game mechanics.
> > Fainted Pokemon should not be able to perform field actions or follow the player. I know that fainted Pokemon can do this in the main series games, but that never really made sense to me, and given that this is a survival/exploration game that kind of does its own thing, I think disabling all actions for a fainted Pokemon would be more realistic and a refreshing twist on the standard game mechanics. In particular it would give the player a lot more incentive to strategize, make use of the sleeping bag, and make safe clearings here and there to protect themselves.
> > At the very least, this should be an option players can select while starting the game, for added difficulty.
> 
> Agreed, it makes the game much more interesting as well, just having fainted mon and using them like nothing takes the fun out of it.

The one possible complexity involved here is, how to handle when a Pokemon that you're currently riding or surfing on faints?

Here's what I'd do. If a Pokemon you're riding faints, that's simple, you'll just be on foot when the battle is over. If a Pokemon you're surfing on faints, then if you have another active Pokemon that can surf, you'll just automatically be surfing on it when the battle is over. If not, then if you have an active Pokemon that can fly, you'll automatically be flying on it when the battle is over. If there are no Pokemon with surf or fly, then you will automatically flee from battle with the message "[Player] has no Pokemon that can surf! [Player] used Escape Rope to return to the shore!" and basically just automatically use Escape Rope to leave the water.
